### ISSUE TITLE
Autocorrect generation for `Naming/MethodName` does not yield a graceful 0 errors rubocop execution

### ISSUE THREAD

**Issue by luke-hill** [rubocop/rubocop#13795]
--------

## Expected behavior

After running `rubocop --auto-gen-config` any offenses for `Naming/MethodName` should be disregarded and not yield further offenses on subsequent `rubocop` executions

## Actual behavior

<details>
[13:51:19] luke@luke-Dell-G15-5530:~/Code/gherkin/ruby$ rubocop --debug
/usr/share/rvm/gems/ruby-3.3.6/gems/rubocop-1.26.1/lib/rubocop.rb:670: warning: base64 was loaded from the standard library, but will no longer be part of the default gems starting from Ruby 3.4.0.
You can add base64 to your Gemfile or gemspec to silence this warning.
For /home/luke/Code/gherkin/ruby: configuration from /home/luke/Code/gherkin/ruby/.rubocop.yml
configuration from /usr/share/rvm/gems/ruby-3.3.6/gems/rubocop-packaging-0.5.1/config/default.yml
configuration from /usr/share/rvm/gems/ruby-3.3.6/gems/rubocop-packaging-0.5.1/config/default.yml
Default configuration from /usr/share/rvm/gems/ruby-3.3.6/gems/rubocop-1.26.1/config/default.yml
configuration from /usr/share/rvm/gems/ruby-3.3.6/gems/rubocop-performance-1.5.2/config/default.yml
configuration from /usr/share/rvm/gems/ruby-3.3.6/gems/rubocop-performance-1.5.2/config/default.yml
configuration from /usr/share/rvm/gems/ruby-3.3.6/gems/rubocop-rake-0.6.0/config/default.yml
configuration from /usr/share/rvm/gems/ruby-3.3.6/gems/rubocop-rake-0.6.0/config/default.yml
configuration from /usr/share/rvm/gems/ruby-3.3.6/gems/rubocop-rspec-2.6.0/config/default.yml
configuration from /usr/share/rvm/gems/ruby-3.3.6/gems/rubocop-rspec-2.6.0/config/default.yml
Inheriting configuration from /home/luke/Code/gherkin/ruby/.rubocop_todo.yml
Use parallel by default.
Running parallel inspection
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/7d19bbdc9efc0bc9067388bc5bb928ce77145523
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/0058c7304ad21fcc5744ede46d65c38b112f4b8b
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/a646362ea38eb5ab8104737abaa950c7d664a902
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/67262a8f3dbbcaabfbbbc69166fb652fcbaf8e75
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/8671515f79b11a103f73abe0c93fac659a29141b
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/d5cf66d7e3a7c100d07fee544d468028357d266b
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/ad260faa4f5ba65a775c6f470c8c511fd5ac855c
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/2014bb75d24d4044e12abd8dc8c671b74d3da11e
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/3b3020675d1c1641e8b8e9e4d152d6efc2ad711f
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/01842c6526b5a0fd0d1f1430cac87ebdbcb158b5
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/766e8de19e21de6a5bbbb9d773f3fdad38622a3c
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/d8fc9e593c2255d49c17a67a337d098d12742a24
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/cbdee6bd2bf7d9c341fad0d296ac7e8ca996fee1
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/4f225621206e90d7b1d7331899bf1ba7abd49ef5
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/2dca4904d40762e4725d75f57c9327f15c77903c
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/61255edbb4ebbbd8a9830422293e015686461535
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/d2bc7e903bb4de212ac0752777593d3e2dbe4ba4
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/c1c03f742ee1815f70683320e1e6a176ddd6e0fc
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/879eb6b075c3348f77e0081511821172d5b6bb8b
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/6fa257097b90c5413397e0129c2d7f83fa13f09b
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/1dd1fa2046ea7c365fbe95efa5be94fa4caab043
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/331f1b7b7ddcc704f022ad41a78756b8c4977ec7
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/2b2a27fe5d90117e3bdb95f6121382819d8fa2f6
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/c510666d0a6dce71bfb44e74c4ab4b2d7cde91e8
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/fafdbae3a533cd0b889987b3a80175506ff7e7bb
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/0d4ee6e57454d95c8f75aadab2fea70cb0b36988
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/af7e7eea3160dca47bd2dcf5dd96f8f73c232244
Inspecting 27 files
Scanning /home/luke/Code/gherkin/ruby/Gemfile
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/2b2a27fe5d90117e3bdb95f6121382819d8fa2f6
.Scanning /home/luke/Code/gherkin/ruby/Rakefile
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/331f1b7b7ddcc704f022ad41a78756b8c4977ec7
.Scanning /home/luke/Code/gherkin/ruby/bin/gherkin
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/ad260faa4f5ba65a775c6f470c8c511fd5ac855c
.Scanning /home/luke/Code/gherkin/ruby/bin/gherkin-generate-tokens
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/6fa257097b90c5413397e0129c2d7f83fa13f09b
.Scanning /home/luke/Code/gherkin/ruby/bin/gherkin-ruby
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/af7e7eea3160dca47bd2dcf5dd96f8f73c232244
.Scanning /home/luke/Code/gherkin/ruby/cucumber-gherkin.gemspec
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/0d4ee6e57454d95c8f75aadab2fea70cb0b36988
.Scanning /home/luke/Code/gherkin/ruby/lib/gherkin.rb
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/1dd1fa2046ea7c365fbe95efa5be94fa4caab043
.Scanning /home/luke/Code/gherkin/ruby/lib/gherkin/ast_builder.rb
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/67262a8f3dbbcaabfbbbc69166fb652fcbaf8e75
.Scanning /home/luke/Code/gherkin/ruby/lib/gherkin/ast_node.rb
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/61255edbb4ebbbd8a9830422293e015686461535
.Scanning /home/luke/Code/gherkin/ruby/lib/gherkin/dialect.rb
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/c1c03f742ee1815f70683320e1e6a176ddd6e0fc
.Scanning /home/luke/Code/gherkin/ruby/lib/gherkin/errors.rb
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/8671515f79b11a103f73abe0c93fac659a29141b
.Scanning /home/luke/Code/gherkin/ruby/lib/gherkin/gherkin_line.rb
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/879eb6b075c3348f77e0081511821172d5b6bb8b
.Scanning /home/luke/Code/gherkin/ruby/lib/gherkin/parser.rb
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/3b3020675d1c1641e8b8e9e4d152d6efc2ad711f
CScanning /home/luke/Code/gherkin/ruby/lib/gherkin/pickles/compiler.rb
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/a646362ea38eb5ab8104737abaa950c7d664a902
.Scanning /home/luke/Code/gherkin/ruby/lib/gherkin/query.rb
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/0058c7304ad21fcc5744ede46d65c38b112f4b8b
.Scanning /home/luke/Code/gherkin/ruby/lib/gherkin/stream/parser_message_stream.rb
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/2014bb75d24d4044e12abd8dc8c671b74d3da11e
.Scanning /home/luke/Code/gherkin/ruby/lib/gherkin/token.rb
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/d5cf66d7e3a7c100d07fee544d468028357d266b
.Scanning /home/luke/Code/gherkin/ruby/lib/gherkin/token_formatter_builder.rb
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/7d19bbdc9efc0bc9067388bc5bb928ce77145523
.Scanning /home/luke/Code/gherkin/ruby/lib/gherkin/token_matcher.rb
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/c510666d0a6dce71bfb44e74c4ab4b2d7cde91e8
CScanning /home/luke/Code/gherkin/ruby/lib/gherkin/token_scanner.rb
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/fafdbae3a533cd0b889987b3a80175506ff7e7bb
.Scanning /home/luke/Code/gherkin/ruby/spec/gherkin/dialect_spec.rb
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/01842c6526b5a0fd0d1f1430cac87ebdbcb158b5
.Scanning /home/luke/Code/gherkin/ruby/spec/gherkin/gherkin_line_spec.rb
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/766e8de19e21de6a5bbbb9d773f3fdad38622a3c
.Scanning /home/luke/Code/gherkin/ruby/spec/gherkin/gherkin_spec.rb
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/d8fc9e593c2255d49c17a67a337d098d12742a24
.Scanning /home/luke/Code/gherkin/ruby/spec/gherkin/parser_spec.rb
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/cbdee6bd2bf7d9c341fad0d296ac7e8ca996fee1
.Scanning /home/luke/Code/gherkin/ruby/spec/gherkin/query_spec.rb
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/4f225621206e90d7b1d7331899bf1ba7abd49ef5
.Scanning /home/luke/Code/gherkin/ruby/spec/gherkin/stream/parser_message_stream_spec.rb
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/2dca4904d40762e4725d75f57c9327f15c77903c
.Scanning /home/luke/Code/gherkin/ruby/spec/spec_helper.rb
Loading cache from /home/luke/.cache/rubocop_cache/c81c7212f2395c1426192acc36a9a7390ed4b482/6d7a3b621ca1730e04accd938619e4bdab66cfb1/d2bc7e903bb4de212ac0752777593d3e2dbe4ba4
.

Offenses:

lib/gherkin/parser.rb:122:9: C: Naming/MethodName: Use snake_case for method names.
    def match_EOF( context, token)
        ^^^^^^^^^
lib/gherkin/parser.rb:128:9: C: Naming/MethodName: Use snake_case for method names.
    def match_Empty( context, token)
        ^^^^^^^^^^^
lib/gherkin/parser.rb:135:9: C: Naming/MethodName: Use snake_case for method names.
    def match_Comment( context, token)
        ^^^^^^^^^^^^^
lib/gherkin/parser.rb:142:9: C: Naming/MethodName: Use snake_case for method names.
    def match_TagLine( context, token)
        ^^^^^^^^^^^^^
lib/gherkin/parser.rb:149:9: C: Naming/MethodName: Use snake_case for method names.
    def match_FeatureLine( context, token)
        ^^^^^^^^^^^^^^^^^
lib/gherkin/parser.rb:156:9: C: Naming/MethodName: Use snake_case for method names.
    def match_RuleLine( context, token)
        ^^^^^^^^^^^^^^
lib/gherkin/parser.rb:163:9: C: Naming/MethodName: Use snake_case for method names.
    def match_BackgroundLine( context, token)
        ^^^^^^^^^^^^^^^^^^^^
lib/gherkin/parser.rb:170:9: C: Naming/MethodName: Use snake_case for method names.
    def match_ScenarioLine( context, token)
        ^^^^^^^^^^^^^^^^^^
lib/gherkin/parser.rb:177:9: C: Naming/MethodName: Use snake_case for method names.
    def match_ExamplesLine( context, token)
        ^^^^^^^^^^^^^^^^^^
lib/gherkin/parser.rb:184:9: C: Naming/MethodName: Use snake_case for method names.
    def match_StepLine( context, token)
        ^^^^^^^^^^^^^^
lib/gherkin/parser.rb:191:9: C: Naming/MethodName: Use snake_case for method names.
    def match_DocStringSeparator( context, token)
        ^^^^^^^^^^^^^^^^^^^^^^^^
lib/gherkin/parser.rb:198:9: C: Naming/MethodName: Use snake_case for method names.
    def match_TableRow( context, token)
        ^^^^^^^^^^^^^^
lib/gherkin/parser.rb:205:9: C: Naming/MethodName: Use snake_case for method names.
    def match_Language( context, token)
        ^^^^^^^^^^^^^^
lib/gherkin/parser.rb:212:9: C: Naming/MethodName: Use snake_case for method names.
    def match_Other( context, token)
        ^^^^^^^^^^^
lib/gherkin/token_matcher.rb:23:9: C: Naming/MethodName: Use snake_case for method names.
    def match_TagLine(token)
        ^^^^^^^^^^^^^
lib/gherkin/token_matcher.rb:30:9: C: Naming/MethodName: Use snake_case for method names.
    def match_FeatureLine(token)
        ^^^^^^^^^^^^^^^^^
lib/gherkin/token_matcher.rb:34:9: C: Naming/MethodName: Use snake_case for method names.
    def match_RuleLine(token)
        ^^^^^^^^^^^^^^
lib/gherkin/token_matcher.rb:38:9: C: Naming/MethodName: Use snake_case for method names.
    def match_ScenarioLine(token)
        ^^^^^^^^^^^^^^^^^^
lib/gherkin/token_matcher.rb:43:9: C: Naming/MethodName: Use snake_case for method names.
    def match_BackgroundLine(token)
        ^^^^^^^^^^^^^^^^^^^^
lib/gherkin/token_matcher.rb:47:9: C: Naming/MethodName: Use snake_case for method names.
    def match_ExamplesLine(token)
        ^^^^^^^^^^^^^^^^^^
lib/gherkin/token_matcher.rb:51:9: C: Naming/MethodName: Use snake_case for method names.
    def match_TableRow(token)
        ^^^^^^^^^^^^^^
lib/gherkin/token_matcher.rb:58:9: C: Naming/MethodName: Use snake_case for method names.
    def match_Empty(token)
        ^^^^^^^^^^^
lib/gherkin/token_matcher.rb:65:9: C: Naming/MethodName: Use snake_case for method names.
    def match_Comment(token)
        ^^^^^^^^^^^^^
lib/gherkin/token_matcher.rb:73:9: C: Naming/MethodName: Use snake_case for method names.
    def match_Language(token)
        ^^^^^^^^^^^^^^
lib/gherkin/token_matcher.rb:84:9: C: Naming/MethodName: Use snake_case for method names.
    def match_DocStringSeparator(token)
        ^^^^^^^^^^^^^^^^^^^^^^^^
lib/gherkin/token_matcher.rb:95:9: C: Naming/MethodName: Use snake_case for method names.
    def _match_DocStringSeparator(token, separator, is_open)
        ^^^^^^^^^^^^^^^^^^^^^^^^^
lib/gherkin/token_matcher.rb:112:9: C: Naming/MethodName: Use snake_case for method names.
    def match_EOF(token)
        ^^^^^^^^^
lib/gherkin/token_matcher.rb:119:9: C: Naming/MethodName: Use snake_case for method names.
    def match_Other(token)
        ^^^^^^^^^^^
lib/gherkin/token_matcher.rb:125:9: C: Naming/MethodName: Use snake_case for method names.
    def match_StepLine(token)
        ^^^^^^^^^^^^^^

27 files inspected, 29 offenses detected
Finished in 0.24249330899328925 seconds
</details>

## Steps to reproduce the problem

In the gherkin repo for ruby
```
rubocop --auto-gen-config
rubocop
```

Observe in the autogen config there is a line
```
# Offense count: 29
# Configuration parameters: IgnoredPatterns.
# SupportedStyles: snake_case, camelCase
Naming/MethodName:
  EnforcedStyle: snake_case
```

Maybe changing this to `Enabled: false` is better - or providing a filepath of each file that has the issue?

## RuboCop version

Yes this is old. But I'm 99% certain this isn't fixed in later versions

```
/usr/share/rvm/gems/ruby-3.3.6/gems/rubocop-1.26.1/lib/rubocop.rb:670: warning: base64 was loaded from the standard library, but will no longer be part of the default gems starting from Ruby 3.4.0.
You can add base64 to your Gemfile or gemspec to silence this warning.
1.26.1 (using Parser 3.3.7.1, rubocop-ast 1.38.0, running on ruby 3.3.6 x86_64-linux)
  - rubocop-packaging 0.5.1
  - rubocop-performance 1.5.2
  - rubocop-rake 0.6.0
  - rubocop-rspec 2.6.0
```

**Comment by luke-hill** [comment]
Upgrading to rubocop 1.71 then yielded 1000's of warnings / errors. I will try to make heads and tails of them. But it's now becoming quite tricky to manage

**Comment by dvandersluis** [comment]
Hi @luke-hill, I tried running rubocop 1.71.2 on the master branch of https://github.com/cucumber/gherkin (after removing the disable of `Naming/MethodName`), and got this in the .rubocop_todo.yml file:

```yaml
# Offense count: 29
# Configuration parameters: EnforcedStyle, AllowedPatterns.
# SupportedStyles: snake_case, camelCase
Naming/MethodName:
  Exclude:
    - 'lib/gherkin/parser.rb'
    - 'lib/gherkin/token_matcher.rb'
```

So it does seem to be working to me. I'm going to close the issue but feel free to let me know if you still have an issue.

**Comment by dvandersluis** [comment]
No worries, happy to help!
